automation/project/Settings.scala

```diff
@@ -27,7 +27,7 @@ object Settings {
   )
 
   val testSettings = List(
-    testOptions in Test += Tests.Argument("-oF", "-u", "test-reports")
+    testOptions in Test += Tests.Argument("-oDFG", "-u", "test-reports")
```

Comment:
`-D` to show test duration in log.
`-G` to show failed tests and their stacktrace again at the end of log.